specs/001-si-richiede-di/contracts/target-os.md

```
@@ -0,0 +1,70 @@
+# Contract: Target OS
+
+## Overview
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)